pipework

```diff
@@ -230,15 +235,29 @@ ln -s "/proc/$NSPID/ns/net" "/var/run/netns/$NSPID"
 MTU=$(ip link show "$IFNAME" | awk '{print $5}')
 # If it's a bridge, we need to create a veth pair
 [ "$IFTYPE" = bridge ] && {
-  LOCAL_IFNAME="v${CONTAINER_IFNAME}pl${NSPID}"
+  if [ -z "$LOCAL_IFNAME" ]; then
+    LOCAL_IFNAME="v${CONTAINER_IFNAME}pl${NSPID}"
+  fi
   GUEST_IFNAME="v${CONTAINER_IFNAME}pg${NSPID}"
+  # Does the link already exist?
+  if ip link show "$LOCAL_IFNAME" >/dev/null 2>&1; then
+    # link exists, is it in use?
+    if ip link show "$LOCAL_IFNAME" up | grep -q "UP"; then
```

Comment:
If the interface exists but is down it will be removed at line 250 below and re-added later (at line 252)